hyperion/playground/inspect_consistency.py

```
@@ -0,0 +1,352 @@
+                    path=str(relative_path),
+                    content=content
+                ))
+            except (UnicodeDecodeError, PermissionError):
+                # Binary file or permission issue
+                try:
+                    size = file_path.stat().st_size
+                    files.append(RepositoryFile(
+                        path=str(relative_path),
+                        content=f"<binary file - {size} bytes>"
+                    ))
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)